Classes/DDLogMacros.h

```diff
@@ -24,7 +24,11 @@
  * The constant/variable/method responsible for controlling the current log level.
  **/
 #ifndef LOG_LEVEL_DEF
-    #define LOG_LEVEL_DEF ddLogLevel
+    #ifdef ddLogLevel
```

Comment:
Also noticed that this check never works as usually `ddLogLevel` is defined as a variable instead of a preprocessor `#define`.